autoprotocol/protocol.py

```diff
@@ -7896,7 +7896,7 @@ def informatics_helper(informatics, dest, multi_src):
 
         # apply tip types to transfer methods
         for vol, met in zip(volume, method):
-            if not met.tip_type:
+            if met._has_calibration() and not met.tip_type:
```

Comment:
Curious, prior to this change, was a runtime error being raised?

Reply:
Yep! Removing this is what introduced us setting filtered1000s in `mode_params`.

Reply:
The reason I ask is because there are tests in Txappy that depend on a runtime error being triggered so if we gate this with the `_has_calibration() method we may break the tests in Txappy. Here is the line to confirm and update if this is True: https://github.com/strateos/txappy/blob/751890f8a6353d385bc35177490d46cd4a1b7372/test/liquid_handle_test.py#L102

Reply:
Will address the txappy tests and update them in a different PR.